plugins/BEdita/Core/src/Model/Action/GetObjectAction.php

```diff
@@ -74,6 +74,11 @@ public function execute(array $data = [])
         if (Configure::check('Status.level')) {
             $query = $query->find('status', [Configure::read('Status.level')]);
         }
+        if (!empty($data['lang'])) {
+            $query->contain('Translations', function ($q) use ($data) {
```

Comment:
Since this is being repeated in two places already, one might add a `translations` finder to Objects table:

```php
protected function findTranslations(Query $query, array $options)
{
    return $query->contain('Translations', function (Query $query) use ($options) {
        return $query->where(['Translations.lang' => $options['lang']]);
    });
}
```

Therefore, here and in `ListObjectsAction` would become:
```php
$query = $query->find('translations', ['lang' => $data['lang']]);
```

Reply:
Done!